opg-php-fpm-71-ppa-1604/Dockerfile

```diff
@@ -27,49 +50,16 @@ RUN apt-get update -y && \
     curl -sS https://getcomposer.org/installer | php && \
     mv composer.phar /usr/local/bin/composer && \
     rm -f /etc/php/7.1/fpm/php-fpm.conf /etc/php/7.1fpm/pool.d/www.conf /etc/cron.d/php && \
-    mkdir -p /data/session
-
-ADD  docker/confd /etc/confd
-
-ADD  docker/service/php-fpm /etc/sv/php-fpm
-RUN  chmod a+x /etc/sv/php-fpm/run && \
-     ln -s /etc/sv/php-fpm /etc/service/ && \
-     echo "<?php echo phpinfo(); ?>" > /app/public/info.php && \
-     chown app /app/public/info.php
-
-ADD  docker/logrotate.d/app /etc/logrotate.d/app
-
-ADD  docker/my_init.d /etc/my_init.d
-RUN  chmod a+x /etc/my_init.d/*
-
-ADD  docker/php/* /etc/php/7.1/
-
-# disable php-xdebug by default
-RUN  rm -f /etc/php/7.1/fpm/conf.d/*xdebug* && \
-     rm -f /etc/php/7.1/cli/conf.d/*xdebug* && \
-     rm /etc/php/7.1/mods-available/xdebug.ini
-
-ADD  docker/nginx /etc/nginx
-
-RUN  pip install schedule statsd requests && \
-     curl -s https://raw.githubusercontent.com/ministryofjustice/php-fpm-stats-collector/master/php-fpm-stats-collector.py > /usr/local/bin/php-fpm-stats-collector.py && \
-     chmod a+x /usr/local/bin/php-fpm-stats-collector.py
-
-ADD  docker/service/php-fpm-stats /etc/sv/php-fpm-stats
-RUN  chmod a+x /etc/sv/php-fpm-stats/run && ln -s /etc/sv/php-fpm-stats /etc/service/
-
-ENV  STATSD_HOST monitoring
-ENV  STATSD_PORT 8125
-ENV  STATSD_DELAY 10
-ENV  FPM_STATUS_URL http://localhost:81/status
-ENV  OPG_STACKNAME php-fpm
-
-ENV  OPG_PHP_POOL_USER app
-ENV  OPG_PHP_POOL_GROUP app
-ENV  OPG_PHP_POOL_CHILDREN_MAX 4
-ENV  OPG_PHP_POOL_REQUESTS_MAX 0
-ENV  OPG_PHP_POOL_REQUEST_TERMINATE_TIMEOUT 60
-ENV  OPG_PHP_POOL_MEMORY_LIMIT 256M
-ENV  OPG_PHP_POOL_POST_MAX_SIZE 64M
-ENV  OPG_PHP_POOL_UPLOAD_MAX_FILESIZE 64M
-ENV  OPG_PHP_POOL_MAX_INPUT_VARS 1000
+    mkdir -p /data/session && \
```

Comment:
Any chance of some comments here.

Reply:
Sure, I can try to find out what was its purpose. 
Please note, the last comment is just relocating existing directives and shouldn't add/remove anything

Reply:
Are we interested on commnet for line 53 ^ only or in general more description about entire RUN's commands?

Reply:
Generally about the RUN commands. Do what you can.